hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -293,42 +281,25 @@ public synchronized boolean tryLock() {
 
     Pair<LockGetResult, StorageLockFile> latestLock = this.lockService.readCurrentLockFile();
     if (latestLock.getLeft() == LockGetResult.UNKNOWN_ERROR) {
-      logger.info(
-          LOCK_STATE_LOGGER_MSG_WITH_INFO,
-          ownerId,
-          lockFilePath,
-          Thread.currentThread(),
-          FAILED_TO_ACQUIRE,
-          "Failed to get the latest lock status");
+      logInfoLockState(FAILED_TO_ACQUIRE, "Failed to get the latest lock status");
       // We were not able to determine whether a lock was present.
       return false;
     }
 
     if (latestLock.getLeft() == LockGetResult.SUCCESS && isLockStillValid(latestLock.getRight())) {
       String msg = String.format("Lock already held by %s", latestLock.getRight().getOwner());
       // Lock held by others.
```

Comment:
should we validate the holder UUID here?

Reply:
Every time we set the lock `this.setLock`, we validate the holder UUID.